src/controllers/cohort.js

```diff
@@ -1,12 +1,20 @@
 import { createCohort } from '../domain/cohort.js'
 import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
+import * as validation from '../utils/validationFunctions.js'
 
 export const create = async (req, res) => {
+  const { startDate, endDate } = req.body
+
   try {
-    const createdCohort = await createCohort()
+    const { parsedStartDate, parsedEndDate } = validation.dateValidation(
+      startDate,
+      endDate
+    )
+    const createdCohort = await createCohort(parsedStartDate, parsedEndDate)
 
     return sendDataResponse(res, 201, createdCohort)
   } catch (e) {
-    return sendMessageResponse(res, 500, 'Unable to create cohort')
+    console.error(e)
+    return sendMessageResponse(res, 400, { error: e.message })
```

Comment:
Should all errors be responding with status 400? More error handling here seems necessary to ensure we are sending the correct error response. If it's an error we are not expecting, we should return status 500